src/openrct2-ui/input/Shortcuts.cpp

```diff
@@ -638,7 +638,7 @@ static void ShortcutConstructionTurnLeft()
     rct_window* window = window_find_by_class(WC_FOOTPATH);
     if (window != nullptr)
     {
-        window_footpath_keyboard_shortcut_turn_left();
+        window->OnShortcutPressed(ShortcutId::WindowRideConstructionTurnLeft);
```

Comment:
It would be better to leave the refactoring to a separate PR. Have a look at the other shortcuts as we have already solved this problem.

Reply:
Right, I'm back to using the old methods with static cast inside the Shortcuts.cpp file.
It seems a bit weird to do that though, now that we have a class with the shortcut methods already inside it.